tests/particle_stratified_stable_layer/setup_vals.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
```

Comment:
~~We use `#!/usr/bin/env python3` everywhere~~
This is just a python *module* (i.e. only imported) so no need for a header (should have been python3 anyway).